wearipedia/devices/strava/strava.py

```diff
@@ -104,6 +114,11 @@ def _filter_synthetic(self, data, data_type, params):
         # but index into it based on the params. Specifically, we
         # want to return the data between the start and end dates.
 
+        stream_data = set(["heartrate"])
+
+        if data_type in stream_data:
```

Comment:
All data should be filtered - it looks here that the stream data is directly returned without the filter